pkgs/apps/vmd/binary.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv, lib, requireFile, makeWrapper, writeScriptBin, bash, perl, tcl-8_5, tk-8_5, symlinkJoin
+, netcdf, libGLU, xorg, fltk, vrpn, flex, bison, libGL_driver, cudatoolkit, autoPatchelfHook
+}:
+assert
+  lib.asserts.assertMsg
+  (stdenv.isLinux && stdenv.isx86_64)
+  "The VMD binaries with require an x86_64 linux OS with CUDA support.";
+
+let
+  homepage = "https://www.ks.uiuc.edu/Research/vmd/";
+  cudatoolkit_joined = symlinkJoin {
```

Comment:
This doesn't work with the separate inputs directly in `buildInputs`?

Reply:
It does :)